plugins/woo-ai/src/image-background-removal/background-removal-link.tsx

```diff
@@ -29,14 +31,24 @@ const getErrorMessage = ( errorCode?: string ) => {
 	}
 };
 
+const recordBgRemovalTracks = recordTracksFactory(
```

Comment:
This might conflict with this PR (https://github.com/woocommerce/woocommerce/pull/40607). Just something to be mindful of when either of these PRs gets merged but maybe you could preemptively rebase on top of that branch so we use the same code to record tracks.

Reply:
Yup, good call! I was thinking I would just wait for your changes and rebase on top of them 👍🏻